src/tests/specs/smoke/smokeTest.spec.ts

```diff
@@ -122,27 +124,32 @@ describe('Planner Smoke Tests:', () => {
     expect(wiTypes.length).toBe(2);
     expect(wiTypes[0]).toBe('Feature');
     expect(wiTypes[1]).toBe('Bug');
+    await planner.sidePanel.clickScenarios();    
   });
 
   it('Edit Comment and Save', async() => {
+    await planner.sidePanel.clickScenarios();    
     await planner.createWorkItem(c.newWorkItem3);
     expect(await planner.workItemList.hasWorkItem(c.newWorkItem3.title)).toBeTruthy();
     await planner.workItemList.clickWorkItem(c.newWorkItem3.title);
     await planner.quickPreview.addCommentAndSave(c.comment);
-    expect(await planner.quickPreview.hasComment(c.comment)).toBeTruthy();
+    await planner.quickPreview.close();    
   });
 
   it('Edit Comment and Cancel', async() => {
+    await planner.sidePanel.clickScenarios();        
```

Comment:
Instead of repeating the same code at multiple places, I'd suggest you consider the combined use of beforeAll and beforeEach